src/autorange.jl

```diff
@@ -0,0 +1,59 @@
+function autorange(img, tform)
+    R = CartesianRange(indices(img))
+    I = first(R)
+    mn = mx = x = tform(SVector(I))
```

Comment:
unsure. I just assumed there was a reason for it (since it was done explicitly before my change). I just collapsed two lines into one